### ISSUE TITLE
**ALL COMPONENTS** 2.4.7 Focus Visible - Keyboard focus not visible on any of the icons in the page

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9709]
As a user using the keyboard to navigate the left hand navigation bar, tabbing into the left-hand navigation and through each link should be clearly signposted by a focus indicator.  

**Linked work item:** 
#9667 

**Expected Result:**
Keyboard focus should be clearly identified by either a bold-weighted box around each icon in a clearly contrasting colour or by the background of the icon itself changing colour to a clearly contrasting colour (whichever is easiest to achieve).

**Actual Result:**
No keyboard focus indicator exists, meaning that the user has no visual cue as to their position in the page.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/b4f6e401-dc67-4388-82e3-4cf16f023b15)

(Evidence shows focus on search and pagination as an example)

**Comment by johnatawnclementawn** [comment]
Resolved in #9735

**Comment by imogenroberts** [comment]
RETEST: 2.4.7 Focus Visible - Focus visibility much improved by the inclusion of a black and yellow bold-weighted box as a focus indicator. However, focus is still not visible in some areas of the site: after the user has navigated out of the search field (selecting here takes the user back into the search field); and after the user has navigated away from the 'Clear Filters' button (it's assumed that this is where the results sorting combo-box is supposed to receive focus).

**Evidence:**


![Image](https://github.com/archesproject/arches/assets/77161085/855f648b-8971-4683-a57c-50b04f0c17f0)

**Comment by imogenroberts** [comment]
Separate bug raised for the lack of focus indicator on the Sort Filter #9994
